frontend/components/SetupPage/Create/hooks/useGetBridgeRequirementsParams.ts

```diff
@@ -72,7 +72,7 @@ const useGetBridgeRequirementsParamsWithMonthlyGasEstimate = (
       // monthly_gas_estimate
       const monthlyGasEstimate =
         SERVICE_TEMPLATES.find(
-          (template) => template.home_chain === toMiddlewareChain,
+          (template) => template.agentType === selectedAgentType,
```

Comment:
let's check homeChain as well, for the same agent on different chains (e.g. babydegen on base and optimism) the reqs may differ